STYLE.md

```diff
@@ -1,96 +0,0 @@
-# Style Guide 
-
-Rather than write a full style guide, please follow by examples you see in the
-code. If you send me a patch, I will not reject it for style reasons (but I
-will fix it before it gets committed).
-
-## Logging
-
-We support logging structured data, so please do that.
-
-Rather than this:
-
-    @logger.info("Some error occured in request #{request} on input #{input} from client #{ip}")
-
-Do this:
-    
-    @logger.info("Some error occured in this request", :request => request, :input => input, :client => ip)
-
-## Code Style
-
-* comment everything you can think of.
-* indentation: 2 spaces
-* between methods: 1 line
-* sort your requires
-* long lines should wrap at 80 characters. If you wrap at an operator ('or',
-  '+', etc) start the next line with that operator.
-* parentheses on function definitions/calls
-* explicit is better than implicit
-  * implicit returns are forbidden except in the case of a single expression 
-
-The point is consistency and documentation. If you see inconsistencies, let me
-know, and I'll fix them :)
-
-Short example:
-
-      require "something from a gem" # from gem 'thing'
```

Comment:
I do like this notation, but maybe it's not needed. The intent is to be very clear as to what gem provides a given library.